Tests/js/views/services/assets/ez-subitemboxviewservice-tests.js

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-subitemboxviewservice-tests', function (Y) {
+    var serviceTest,
+        Assert = Y.Assert, Mock = Y.Mock;
+
+    serviceTest = new Y.Test.Case({
+        name: "eZ Subitem Box View Service tests",
+
+        setUp: function () {
+            this.content = {};
+            this.contentType = {};
+            this.location = {};
+            this.searchResultList = [{content: this.content, contentType: this.contentType, location: this.location}];
+            this.request = {
+                params: {
+                    searchString: this.searchString,
+                    limit: this.limit
+                }
+            };
+            this.app = new Y.Base();
+
+            this.service = new Y.eZ.SubitemBoxViewService({
+                request: this.request,
+                app: this.app,
+                response: {},
+            });
+
+            this.service.search = new Mock();
+        },
+
+        tearDown: function () {
+            this.service.destroy();
+            delete this.service;
+        },
+
+        "Should get the view parameters": function () {
+            var params,
+                that = this;
+
+            Mock.expect(this.service.search, {
```

Comment:
that's not needed to test `getViewParameters`. Just set some values in the attributes and check you find the same in `getViewParameters` return value.